src/components/Discussion/Internal/Comment/Header.js

```diff
@@ -294,59 +269,39 @@ export const Header = ({ t, comment, menu, isExpanded, onToggle }) => {
           )}
         </div>
       </div>
-      {onToggle && (
-        <button
-          {...headerActionStyle({ isExpanded })}
-          {...headerActionStyleHover}
-          {...colorScheme.set('color', isExpanded ? 'divider' : 'textSoft')}
-          onClick={onToggle}
-        >
-          {!isExpanded && comments && comments.totalCount > 0 && (
-            <div {...styles.expandCount}>
-              {t.pluralize('styleguide/comment/header/expandCount', {
+      <div {...styles.actionsWrapper} className={styles.actionsWrapper}>
+        {onToggle && (
+          <IconButton
+            invert={true}
+            Icon={isExpanded ? UnfoldLessIcon : UnfoldMoreIcon}
+            onClick={onToggle}
+            label={
+              !isExpanded &&
+              t.pluralize('styleguide/comment/header/expandCount', {
                 count: comments.totalCount + 1
-              })}
-            </div>
-          )}
-          {isExpanded ? <IcCollapse /> : <IcExpand />}
-        </button>
-      )}
-      {menu && (
-        <CalloutMenu Element={MoreIconWithProps} align='right'>
-          {menu}
-        </CalloutMenu>
-      )}
+              })
+            }
+            labelShort={
+              !isExpanded &&
+              t.pluralize('styleguide/comment/header/expandCount', {
+                count: comments.totalCount + 1
+              })
+            }
+            noMargin
+          />
+        )}
+        {menu && (
+          <div>
+            <CalloutMenu
```

Comment:
I would use textSoft as the color for both expand and callout icons. It'll soften it on mobile:
<img width="312" alt="Screenshot 2021-11-26 at 14 45 18" src="https://user-images.githubusercontent.com/20746301/143590354-122c6148-4f17-4912-af98-610755695387.png">